tests/tinderbox_scraper/test_revision_tinderbox.py

```diff
@@ -6,33 +6,30 @@
 
 from mock import patch
```

Comment:
Please change to make use of the pytest-mock module instead. See https://github.com/mozilla/mozdownload/blob/master/tests/try_scraper/test_invalid_revision.py how to do it.